.github/workflows/main.yml

```diff
@@ -5,11 +5,19 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version: [
+          8,
+          11,
+          13,
```

Comment:
14 is the latest Java version. 13 is no longer supported.

Reply:
removed